### PR TITLE
Final group capstone: Detail page

### PR DESCRIPTION
**Pull request by wale-prog**
In this pull request, I have implemented the following: 
- Designed the detail page as stated in the requirement
- There is a reserve button that should lead to the reserve page
- More information about a particular room is displayed

### CODE REVIEW

**Review by sntakirutimana72 — APPROVED** [review]
Hi @wale-prog ,

# Status: APPROVED!!!  :champagne: 🥂

Bravo👏!!! Very impressive design and logic, well done.👍

## Highlight! :green_circle: 

:heavy_check_mark:  Great eye for styling.
:heavy_check_mark:  Good commit messages
:heavy_check_mark:  No linter errors.

## [Optional] suggestions

- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

-----

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack using this handle @sntakirutimana72.

File: src/Pages/Room/roomDetail.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+import axios from 'axios';
+import './RoomDetail.css';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCalendarCheck } from '@fortawesome/free-regular-svg-icons';
+import { faCircleChevronRight } from '@fortawesome/free-solid-svg-icons';
+
+const RoomDetail = () => {
+  const [roomDetail, setRoomDetail] = useState([]);
+  // const [message, setMessage] = useState({});
+  const { id } = useParams();
```
Comment (Chrissiku) [inline src/Pages/Room/roomDetail.js]: - [ _Opional_ ]: Please make sure to remove all unnecessary code from the project.

File: src/redux/RoomSlice.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+import axios from 'axios';
+
+async function fetchRoomDetail(id) {
+  const apiUrl = `http://localhost:3000/api/v1/rooms/${id}`;
+  const response = await axios.get(apiUrl);
+  const responseData = await response.json();
+  return responseData;
+}
+
+export default fetchRoomDetail;
```
Comment (Chrissiku) [inline src/redux/RoomSlice.js]: - The function `fetchRoomDetails()` is created and `exported` but never `imported`. Make sure that you remove this part from your project or interact with a configured Redux reducer and then import it in the `src/Pages/roomDetail.js` file

**Review by Chrissiku — CHANGES_REQUESTED** [review]
### STATUS: CHANGES REQUESTED 🔴 
Hi @wale-prog 

You did a great job so far, congratulation 👏🏼 👏🏼
You still need to work on some issues to go to the next part of this project, but you are almost there!

### Highlight 💪🏻

- ✔️ No linter errors.
- ✔️ Gitflow was correctly used. 
- ✔️Have descriptive commit messages. 

### Requested Changes 🔴 
_Please Check the comments under the review and make all required changes_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


_Cheers and Happy coding!👏👏👏 💻_  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me ( @Chrissiku ) when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Chrissiku — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi @wale-prog 👋

Your project is complete! ✔️ 🏁

Finally, **you did it**! 🎉 👏. Your code was very professional 💯 and I can see you have a great chance to be a Great Software Engineer ⛑️.

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉. And Welcome to Microverse 👋.

> **Note:** You can now take a break even if short before you move on to the next project. Happy 😄 Coding 🖥️! and Have a nice day 🌞.

## Highlights 🌟

- ✔️ Descriptive commmit messsages
- ✔️ Descriptive PR description
- ✔️ No linter Errors
- The code is top-notch! 💯 and have a huge potential.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by safar1212 — APPROVED** [review]
# APPROVED!!!! 🎉
## Hey @wale-prog   😊 😃 

#### Great Job 😄 👏 You've done it. Nice completing this milestone 👍🏻 ✅ 

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by aliatayee — COMMENTED** [review]
# 🟢 Project Approved! 🏆

## Hi @wale-prog 👋

Your project is complete! ✔️ 🏁
There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉.

## Highlights 🌟

- ✔️ Descriptive commmit messsages
- ✔️ Descriptive PR description
- ✔️ No linter Errors
- The code is top-notch! 💯 and have a huge potential.

Cheers and Happy coding!👏👏👏

**Review by aliatayee — APPROVED** [review]

**Pull request merged**